problem_db/class_point.tut/preload.py

```diff
@@ -3,9 +3,18 @@
 
 
 class Point(object):
+    """A 2D point in the cartesian plane"""
     def __init__(self, x, y):
+        """
+        Constructs a point object given the x and y coordinates
```

Comment:
Good point I should be consistent with this. I'll stick with imperative mood. Thanks for the feedback :), I still recall being tutored by you in 2014 SM2....